guides/centering_accuracy.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: guides
+title: Guides | Centring Accuracy 
+description: This guide teaches you how to centre the accuracy on the playfield.
+---
+
+# Centring Accuracy
+
+This guide will teach you how to centre the accuracy counter on the playfield.
+
+Note that by doing this, the percentage symbol after the accuracy will disappear. This is due to osu! stopping to load the texture and just the hitbox, due to its size.
+
+## Roughly centring accuracy
+This part of the guide will teach you how to roughly centre the accuracy counter - i.e. centring it 'good enough'. To learn how to perfectly centre accuracy, look at the `Perfectly centring accuracy` part of this guide.
+
+1. Open your skin's folder. The folder can be found by navigating to the `Skins\` directory of your osu! installation. By default this will be located at `C:\Users\[User Name]\AppData\Local\osu!\Skins\`. You can locate this by pressing `CTRL + O` in the osu! window to bring up settings, clicking `Open osu! folder`, and navigating to the `Skins\` directory. Alternatively, you can go to settings, navigate to the `SKIN` section, and click `Open current skin folder`.
+2. Find your resolution's width. This will be the first number in your resolution setting (e.g. 1920 is the width of the resolution 1920x1080). You can find this by opening up osu! settings, navigating to the `GRAPHICS` section, and viewing the `Resolution:` setting.
+3. Open your `score-percent.png` (or `score-percent@2x.png` for resolutions with a height of 800 or above) file in an image editing program and use the following formula to find the width which the file will need to be:

```

Comment:
iirc all guides say that a basic understanding of HD and SD files is required and link to where to learn about that.

This guide should probably do the same instead of having the bracketing

Reply:
added that at the top, but also changed the bracketting to be like score-percent(@2x).png, is that alright or should we do away with it completely?